core/core.md

```diff
@@ -6,37 +6,37 @@ As mentioned in the README, the Fora protocol relies on ActivityPub run over a d
 
-If the user is a Light User must then send this message to a Host of their choice. If the user is a Host, they simply propogate their message to the rest of their peers.
+If the member is a Light member must then send this message to a Host of their choice. If the member is a Host, they simply propogate their message to the rest of their peers.
 
-Any Host receiving a message must first check that the user it is coming from is a valid member of the community (ie accepted and not banned), and then verify that the signature is valid, and that the content is valid and not banned. If all these checks pass, the Host will apply the message to modify its own internal state and then gossip the message to the rest of its peers.
+Any Host receiving a message must first check that the member it is coming from is a valid member of the community (ie accepted and not banned), and then verify that the signature is valid, and that the content is valid and not banned. If all these checks pass, the Host will apply the message to modify its own internal state and then gossip the message to the rest of its peers.
 
-Hosts themselves will have access to the full state of the community, subject to liveness constraints. However, Light Users must query a Host in order to get the relevant parts of the community state. At any given point, a Host may not have access to the latest community state. If this is the case, the Light User may query other Hosts to fill in the missing information, or they may wait for the Host to query this information itself and update the Light User's feed.
+Hosts themselves will have access to the full state of the community, subject to liveness constraints. However, Light members must query a Host in order to get the relevant parts of the community state. At any given point, a Host may not have access to the latest community state. If this is the case, the Light member may query other Hosts to fill in the missing information, or they may wait for the Host to query this information itself and update the Light member's feed.
 
-Missing information from other users will be sometimes be detectable if there are missing sequences in the user's message history. If this is the case, then Hosts can query for the missing information if it indeed exists. Since malicious users may intentionally create missing sequences, Hosts and Light User implementations should not make the assumption that this message actually exists.
+Missing information from other members will be sometimes be detectable if there are missing sequences in the member's message history. If this is the case, then Hosts can query for the missing information if it indeed exists. Since malicious members may intentionally create missing sequences, Hosts and Light member implementations should not make the assumption that this message actually exists.
```

Comment:
```suggestion
Missing information from other members will be sometimes be detectable if there are missing sequences in the member's message history. If this is the case, then Hosts can query for the missing information if it indeed exists. Since malicious members may intentionally create missing sequences, Hosts and Light User implementations should not make the assumption that this message actually exists.
```